### ISSUE TITLE
Orderby tests broken

### ISSUE THREAD

**Issue by hylkevds** [opengeospatial/ets-sta10#64]
**Describe the bug**
The orderby test creates an invalid URL:
`https://ogc-demo.k8s.ilt-dmz.iosb.fraunhofer.de/FROST-CERTIFY/v1.0/Things?$orderby=name%20desc.  [Request] https://ogc-demo.k8s.ilt-dmz.iosb.fraunhofer.de/FROST-CERTIFY/v1.0/Things?$orderby=name%20asc`
Yes, that entire thing is the URL it generates.

**To Reproduce**
1. Run the test

**Expected behavior**
No invalid URL generated

**Additional context**
The problem is [here](https://github.com/opengeospatial/ets-sta10/blob/fdea15e2da6c45954c92019095cf3697c77d926a/src/main/java/org/opengis/cite/sta10/filteringExtension/Capability3Tests.java#L443)
```
                urlString = ServiceURLBuilder.buildURLString(rootUri, entityType, -1, null, "?$orderby=" + property + "%20desc.  [Request] " + urlString);
```
The previous value of urlString is appended to the new urlString, including the `.  [Request]` bit.

**Comment by hylkevds** [comment]
Also explains why all certifications ran out :)

**Comment by dstenger** [comment]
Thank you for reporting and providing the pull request.
Indeed, this seems to be a bug.

**Comment by hylkevds** [comment]
When do you plan on switching the official test to 1.4? Currently no implementation can pass certification.

**Comment by dstenger** [comment]
We will release 1.4 on [Beta](https://cite.ogc.org/te2/) soon (end of November).
There it will stay for at least one month before we can move it to [Production](https://cite.ogc.org/teamengine/).

**Comment by hylkevds** [comment]
Is there anyone actively testing it while it is in Beta? Since obviously no one tested the v1.3 release...

**Comment by dstenger** [comment]
You are welcome to execute some tests as soon as it is installed on Beta. The release will be announced via the CITE mailing lists.
Also, the SWG can be contacted to ask implementers of the standard to test their implementations on Beta.

**Comment by hylkevds** [comment]
I suspect no one from the STA-SWG in on the CITE mailing list. That would explain why no one tested v1.3 last year. Wouldn't it make more sense to announce the beta release of new version on the mailing list of the relevant SWGs? That way the relevant people are certain to get the notice.

**Comment by danistrigaro** [comment]
> We will release 1.4 on [Beta](https://cite.ogc.org/te2/) soon (end of November). There it will stay for at least one month before we can move it to [Production](https://cite.ogc.org/teamengine/).

Dear, would be nice to have the new version in production, since we spent a lot of time to find out this issue. By the way thank you for reporting and commenting and linking the beta version!
